components/menu/MenuDivider.razor

```diff
@@ -0,0 +1,28 @@
+﻿@namespace AntDesign
+@inherits AntDomComponentBase
+
+<li Class="@ClassMapper.Class"></li>
+
+@code {
+    [CascadingParameter]
+    public Menu RootMenu { get; set; }
+
+    private void setClass()
+    {        
+        ClassMapper.Clear()
+            .Add($"{RootMenu.PrefixCls}-item-divider");            
+    }
+
+    protected override void OnParametersSet()
+    {
+        setClass();
```

Comment:
Only need call setClass in OnInitialized

Reply:
Commit fea40f7